docs/standards/smart-contracts/lsp4-digital-asset-metadata.md

```diff
@@ -43,7 +43,7 @@ constructor(
 ) ERC725Y(newOwner_)
 ```
 
-Sets the name, symbol of the token and **[SupportedStandards:LSP4DigitalAsset](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md#supportedstandardslsp4digitalasset)** Key in the **[ERC725Y Key/Value store](./lsp0-erc725-account#setdata)** and sets the **initial owner** of the contract.
+Sets the token's name, its symbol, and the **[SupportedStandards:LSP4DigitalAsset](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md#supportedstandardslsp4digitalasset)** key on the **[ERC725Y Key-Value Store](./lsp0-erc725-account#setdata)**. It also sets the **initial owner** of the contract.
```

Comment:
What about using a bullet list to describe the constructor here?

```markdown
Sets the **initial owner** of the contract, and the following data keys on the **[ERC725Y Key-Value Store](./lsp0-erc725-account#setdata)** of the contract:

- `name_`: token's name.
- `symbol_`: token's symbol.
```

Reply:
Yes indeed! bullet list would be ideal

Reply:
Should also use the same structure for LSP7 and LSP8 within the same folder